benefits/eligibility/forms.py

```diff
@@ -42,6 +42,33 @@ def clean(self):
             raise forms.ValidationError("reCAPTCHA failed")
 
 
+class InPersonEligibilityForm(forms.Form):
+    """Form to capture enrollment flow selection."""
+
+    action_url = routes.IN_PERSON_ENROLLMENT
+    id = "form-flow-selection"
+    method = "POST"
+
+    flow = forms.ChoiceField(label="Choose an eligibility type to qualify this rider.", widget=forms.widgets.RadioSelect)
+    verified = forms.BooleanField(label="I have verified this person’s eligibility for a transit benefit.", required=True)
+
+    cancel_url = routes.ADMIN_INDEX
+
+    def __init__(self, agency: models.TransitAgency, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        flows = agency.enrollment_flows.all()
+
+        self.classes = "checkbox-parent"
+        flow_field = self.fields["flow"]
+        flow_field.choices = [(f.id, f.label) for f in flows]
```

Comment:
This sets the form field's label as the `label`, which are currently named like this:

```
(CST) Senior Discount
CalFresh
(CST) Veteran Discount
(CST) Agency Card Discount
```

It's slightly different from what we want (Older adult, CalFresh card holder, Courtesy Card holder, etc.) - which are specified in these files for the Benefits app: https://github.com/cal-itp/benefits/blob/1e2ff89cfcdbcddc6895701399aaf1702b3b2fc1/benefits/eligibility/templates/eligibility/includes/selection-label--senior.html

 But didn't want to bog down this PR with implementing all of that.

Reply:
Created separate issue https://github.com/cal-itp/benefits/issues/2336 to cover this